apps/passport-server/migrations/63_generic_issuance_pipelines.sql

```diff
@@ -0,0 +1,23 @@
+
+CREATE TABLE generic_issuance_users (
+  id UUID NOT NULL PRIMARY KEY,
+  email VARCHAR NOT NULL UNIQUE,
+  is_admin BOOLEAN NOT NULL
+);
+
+CREATE TABLE generic_issuance_pipelines (
+  id UUID NOT NULL PRIMARY KEY,
+  owner_user_id UUID NOT NULL REFERENCES generic_issuance_users(id),
+  pipeline_type VARCHAR NOT NULL CHECK(pipeline_type IN ('Lemonade', 'Pretix')),
```

Comment:
Not blocking, but it would be annoying to have to change this for every new pipeline type be add. For example, we're going to have a manual entry / CSV pipeline in a couple weeks which will require a modification here. I think checking on the server-side probably makes more sense.